sdk/web-pubsub/web-pubsub/tests.yml

```diff
@@ -0,0 +1,13 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/web-pubsub"
+      ServiceDirectory: web-pubsub
+      MatrixFilters:
+        - DependencyVersion=^$
```

Comment:
Min-max testing is disabled because the test folder doesn't follow the expected min-max compliant structure